pkg/asset/installconfig/powervs/client.go

```diff
@@ -42,16 +44,18 @@ type API interface {
 	ListServiceInstances(ctx context.Context) ([]string, error)
 	ServiceInstanceGUIDToName(ctx context.Context, id string) (string, error)
 	GetDatacenterCapabilities(ctx context.Context, region string) (map[string]bool, error)
+	GetAttachedTransitGateway(ctx context.Context, svcInsID string) (string, error)
```

Comment:
Don't we also need the `GetTGConnectionVPC` header here?

Reply:
Of course, we do. Thanks!